flask_jwt_extended/tokens.py

```diff
@@ -91,6 +108,8 @@ def encode_refresh_token(identity, secret, algorithm, expires_delta, user_claims
     :type expires_delta: datetime.timedelta or False
     :param user_claims: Custom claims to include in this token. This data must
                         be json serializable
+    :param additional_claims: Custom claims to include in this token. Object
+                        must be json serializable
```

Comment:
Not only must it be json serializable, it must be a dictionary. Should update the wording here to match.